stock_mts_mto_mrp_rule/demo/demo.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo noupdate="1">
+    <record id="stock_location_route_dummy_route" model="stock.location.route">
```

Comment:
It's better to create those 'dummy' data in setupClass of tests if the aim is to use them there only.

In demo, should appear only real 'demo' data (with significant meaning - something you want to show to user).

Reply:
Resolved. Moved demo data to TestCommon